packages/camera/camera_android_camerax/android/src/main/java/io/flutter/plugins/camerax/CameraXProxy.java

```diff
@@ -23,4 +27,17 @@ public DeviceOrientationManager createDeviceOrientationManager(
       DeviceOrientationManager.DeviceOrientationChangeCallback callback) {
     return new DeviceOrientationManager(activity, isFrontFacing, sensorOrientation, callback);
   }
+
+  public Preview.Builder createPreviewBuilder() {
+    return new Preview.Builder();
+  }
+
+  public Surface createSurface(SurfaceTexture surfaceTexture) {
+    return new Surface(surfaceTexture);
+  }
+
+  public SystemServicesFlutterApiImpl createSystemServicesFlutterApi(
```

Comment:
This is an odd name for a method normally an Impl is reserved for an implementation of an interface. 
Consider renaming.